### PR TITLE
Turn class/test

### PR DESCRIPTION
**Pull request by e-spitz**
# Description

Built out passing Turn tests and Turn class. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

# How Has This Been Tested?
Passing terminal tests

- [ ] open index.html
- [ ] dev tools
- [x] terminal

# Checklist:

- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

File: src/Turn.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  returnCard() {
+    return this.currentCard;
+  }
+
+  evaluateGuess() {
+    return this.guess === this.currentCard.correctAnswer;
+  }
+
+  giveFeedback() {
+    return this.evaluateGuess() ? 'Correct!' : 'Incorrect!';
```
Comment (Meekb) [inline src/Turn.js:20]: nice ternary - im so proud!

File: test/Turn-test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    expect(Turn).to.be.a('function');
+  });
+
+  it('should instantiate a new turn', () => {
+    expect(turn).to.be.an.instanceof(Turn);
+  });
+
+  it('should have a user guess', () => {
+    expect(turn.guess).to.equal('array');
+    expect(turn.guess).to.be.a('string');
```
Comment (Meekb) [inline test/Turn-test.js:26]: i don't know if this is me being picky, but i'd move the data type expect  on line 26 to 25  and then check for the answer to 26.

File: test/Turn-test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  it('should have a current card in play', () => {
+    expect(turn.currentCard).to.equal(card);
+    expect(turn.currentCard).to.be.an('object');
```
Comment (Meekb) [inline test/Turn-test.js:31]: same here. i think i might just be picky.

**Review by Meekb — APPROVED** [review]

File: test/Turn-test.js (hunk 2 above)
Comment (e-spitz) [reply test/Turn-test.js:26]: Good call, will do!

File: src/Turn.js (hunk 1 above)
Comment (e-spitz) [reply src/Turn.js:20]: <3

**Pull request merged**